### ISSUE TITLE
Using borb via pip will not import Document class

### ISSUE THREAD

**Issue by nls81** [jorisschellekens/borb#72]
I am creating a PDF and when I try to run the code, it says:
ImportError: cannot import name 'Document' from 'borb.pdf.document'

`
from borb.pdf.document import Document
from borb.pdf.page.page import Page
from borb.pdf.canvas.layout.page_layout.multi_column_layout import SingleColumnLayout
from borb.pdf.canvas.layout.image.image import Image
from borb.pdf.canvas.layout.table.fixed_column_width_table import FixedColumnWidthTable as Table
from borb.pdf.canvas.layout.text.paragraph import Paragraph
from borb.pdf.canvas.layout.layout_element import Alignment
from borb.pdf.pdf import PDF
from decimal import Decimal

def receipt_and_invoice_creator(invoice_number, date):
    # Create document
    pdf = Document()

    # Add page
    page = Page()
    pdf.append_page(page)

    # Make a page layout to manage coordinates
    page_layout = SingleColumnLayout(page)
    page_layout.vertical_margin = page.get_page_info().get_height() * Decimal(0.02)

    # Add image to PDF
    page_layout.add(    
        Image(        
        "Example.com/logo.png",        
        width=Decimal(128),        
        height=Decimal(128),    
    ))

    company_street_address = '123 Happy Place'
    company_city_state_zip = 'New York, NY 12345'
    company_email = 'Support@Example.com'
    company_website = 'Example.Com'
    #Create table for company info
    company_info_table = Table(number_of_rows=4, number_of_columns=3)
    company_info_table.add(Paragraph(company_street_address))    
    company_info_table.add(Paragraph("Date", font="Helvetica-Bold", horizontal_alignment=Alignment.RIGHT))      
    company_info_table.add(Paragraph(date))
	
    company_info_table.add(Paragraph(company_city_state_zip))    
    company_info_table.add(Paragraph("Receipt #", font="Helvetica-Bold", horizontal_alignment=Alignment.RIGHT))
    company_info_table.add(Paragraph(invoice_number))
	
    company_info_table.add(Paragraph(company_email))    
    company_info_table.add(Paragraph(" "))
    company_info_table.add(Paragraph(" "))

    company_info_table.add(Paragraph(company_website))
    company_info_table.add(Paragraph(" "))
    company_info_table.add(Paragraph(" "))

    company_info_table.set_padding_on_all_cells(Decimal(2), Decimal(2), Decimal(2), Decimal(2))    		
    company_info_table.no_borders()

    with open("output.pdf", "wb") as pdf_file_handle:
        PDF.dumps(pdf_file_handle, pdf)

receipt_and_invoice_creator('afe91f03-394d-40c0-8dee-f6a4981c6b5c', 'January 4, 2022')
`

I am using VS Code and it claims that
`from borb.pdf.document import Document` 
does not import anything but
`from borb.pdf.document import document`
does import a class

However, when I change it to that, I get an error:
pdf = document()
TypeError: 'module' object is not callable

And if I get rid of the callable, it tells me:
AttributeError: module 'borb.pdf.document.document' has no attribute 'append_page'

**Comment by nls81** [comment]
For some reason Github is not treating all my code as code despite being in proper markdown - but still should make sense

**Comment by jorisschellekens** [comment]
This is not a `pip` error, nor is it related to `borb`.

As mentioned in the release-notes, I moved quite a bit of logic into its own separate classes. Notable mentions are `NameTree` and `Annotation` (and all its subclasses). 

By doing so, I sometimes ended up with a new package. For instance, previously `Document` did not have a package. But now that there are at least two classes at this level (`NameTree` and `Document`) it made sense to create a package.

I explicitly mentioned (in the release-notes) that you would do well to check your imports.

I also made sure the "Hello World" example in the readme is updated to ensure it uses the new imports. And I am currently in the process of updating all other documentation.

Short summary: Read the documentation and release-notes before creating an issue. And fix your imports to ensure your code runs smoothly. Using an IDE can assist you.

Kind regards,
Joris Schellekens
